llvm/lib/SYCLLowerIR/SpecConstants.cpp

```diff
@@ -827,6 +840,9 @@ PreservedAnalyses SpecConstantsPass::run(Module &M,
   MapVector<StringRef, MDNode *> SCMetadata;
   SmallVector<MDNode *, 4> DefaultsMetadata;
 
+  if (SpecConstantMode.getNumOccurrences() > 0)
```

Comment:
What if user specifies an unexpected value for this option? That might need to be captured somehow.

Thanks